tests/test_interactive.py

```diff
@@ -96,6 +96,13 @@ def test_sdl_audio_beeps(self):
     Popen([PYTHON, EMCC, '-O2', '--closure', '1', '--minify', '0', os.path.join(self.get_dir(), 'sdl_audio_beep.cpp'), '-s', 'DISABLE_EXCEPTION_CATCHING=0', '-o', 'page.html']).communicate()
     self.run_browser('page.html', '', '/report_result?1')
 
+  def test_sdl2_mixer(self):
```

Comment:
I have a question here - what's the point of test_interactive? It seems most of the tests automatically complete (very like the test_browser tests), so they don't seem to be human verified?

I kinda assumed that test_interactive was a bit like test_browser (in that you run that suite always and verify it passes) but you can test it 'more' if you feel like it by running the test manually.

Reply:
`test_interactive.py` contains tests that require human interaction to complete, i.e. things that cannot be automatically verified, such as making sure that audio actually comes out from the speakers when running an audio test. Many of the interactive tests are duplicated in `test_browser.py` but in automated form, so that their successful compilation is also automatically tested.

Reply:
As a concrete example, I believe `test_sdl_audio` terminated automatically when I was trying it out. But I see the intention now, and looking at the code of some of the others I can see how it should work.